packages/office-ui-fabric-react/src/components/Divider/examples/VerticalDivider.Basic.Example.tsx

```diff
@@ -31,15 +31,14 @@ const getExampleClassNames = memoizeFunction(
   },
 );
 
-export class VerticalDividerBasicExample extends React.Component<any, any> {
-  public render(): JSX.Element {
-    const classNames = getExampleClassNames();
-    return (
-      <div className={classNames.wrapper}>
-        <p className={classNames.text}> Some text before the divider. </p>
-        <VerticalDivider />
-        <p className={classNames.text}>Some text after the divider. </p>
-      </div>
-    );
-  }
-}
+export const VerticalDividerBasicExample = () => {
```

Comment:
For all the examples, it's good to explicitly add a type to the component. In this case:
```suggestion
export const VerticalDividerBasicExample: React.FunctionComponent = () => {
```